Lib/fontTools/pens/ttGlyphPen.py

```diff
@@ -127,7 +127,12 @@ def _buildComponents(self, componentFlags):
             components.append(component)
         return components
 
-    def glyph(self, componentFlags: int = 0x04, dropImpliedOnCurves=False) -> Glyph:
+    def glyph(
+        self,
+        componentFlags: int = 0x04,
+        dropImpliedOnCurves: bool = False,
+        roundCoordinates: bool = True,
```

Comment:
We typically just pass a `round=round` where one can pass `round=noRound` from `roundTools`. The code can optimize for `noRound`. I'm fine with what you have though.

Reply:
ah makes sense, thanks for the suggestion, I'll do that